### DISCUSSION TITLE
RASCSI with Sun Sparcstation

### DISCUSSION THREAD

**Discussion by RobPattonDotCom** [akuker/RASCSI#962] [General]
Anyone have any success using this with Sun Sparcstations?  

I connect it and can do a probe-scsi-all and I see any devices I create, but I have yet to be able to boot from an iso/cd, or to be able to format a disk image to install sunos/Solaris.     Would love to know if its been done already or if I'm running into some weird issue I can't see.

**Comment by RobPattonDotCom** [comment]
Bonus points if someone can offer a sunos drive image so I can make some progress

**Comment by guruthree** [comment]
Before the power supply to my Sun Ultra 1 cooked itself, I had the RaSCSI booting both Solaris and Debian. It's been a few months now, so my memory is a bit fuzzy and my notes are a bit vague, but I seem to recall getting Debian working was a lot easier. I think that needed just a 'boot cdrom' command and after that it was a standard Linux install. Getting Solaris to install was a bit more involved, and the drive needed to be initialised before the installer would work. I think this was done using a 'boot cdrom -s' to get into the installer's recovery mode(?), at which point some variety of partitioning/low level format was done. The only link I saved was for the [format](https://docs.oracle.com/cd/E23824_01/html/821-1459/disksprep-10.html) command, but I think there was more to it. I think the SCSI device ID for the CD drive was the RaSCSI default, and for the drive I used the SUN 4.2G preset.
